.travis_build.sh

```diff
@@ -128,7 +128,7 @@ case "$PLATFORM" in
     pushd "${build_directory}"
     # remove bochs plugins
     sed -i 's/Bochs.* //g' plugins.ext
-    make
+	./mvm -f
```

Comment:
Big pull requests are scary, it's sort of take it or leave it as a whole...

To mitigate that, I was trying to review the changes you submitted thru github web interface:
https://github.com/OpenSmalltalk/opensmalltalk-vm/pull/52
Unfortunately, it's not possible (too many changes, diffs are not displayed).

So I tried to pick and review intermediate changes. But then there's the risk to review obsolescent things...
Especially, if there has been several cross-merging (rather than rebase).
Unless there's a massive rewrite this also is a dead end (so forget about above remarks).

Unfortunately, we're back to take it or leave it initial status...